Source/Core/CommandLineOptions.cs

```diff
@@ -1197,6 +1176,14 @@ protected override bool ParseOption(string name, CommandLineOptionEngine.Command
 
           return true;
 
+        case "printPruned":
```

Comment:
Describe this option in the `/help` message.

Also, describe `{:include_dep}` (and delete the description for `{:exclude_dep}`) in the `/attrHelp` message in this file.

Reply:
The description in the comment is ~50 lines long, that doesn't seem to fit well in help. Would a summary of the description be useful?